package.json

```diff
@@ -56,6 +56,7 @@
     "broccoli-middleware": "^2.0.1",
     "broccoli-module-normalizer": "^1.3.0",
     "broccoli-module-unification-reexporter": "^1.0.0",
+    "broccoli-slow-trees": "^3.0.1",
```

Comment:
Shall we expose lib/messages from broccoli and include the slowTrees printing from there? I think that’s better than the dependency here

Reply:
I 100% agree that we should use broccoli's message util. However, we should explore that as a quick follow up so that we can start validating the watcher / broccoli 3 stuff now.

Reply:
Cool, first step done here https://github.com/oligriffiths/broccoli-slow-trees/pull/1/files?w=1